www/src/components/repository/RepositorySideCar.tsx

```diff
@@ -0,0 +1,133 @@
+import { useContext, useState } from 'react'
+import { Link, useLocation } from 'react-router-dom'
+import { A, Div, Flex, Img, P, Button, Icon } from 'honorable'
+import { BrowserIcon, CertificateIcon, GearTrainIcon, GitHubIcon, InvoicesIcon, LinksIcon, Tag } from 'pluralsh-design-system'
+
+
+import RepositoryContext from '../../contexts/RepositoryContext'
+import usePaginatedQuery from '../../hooks/usePaginatedQuery'
+
+import { capitalize } from '../../utils/string'
+
+import { InferredConsoleButton } from '../clusters/ConsoleButton'
+
+import { RECIPES_QUERY } from './queries'
+import { providerToIcon, providerToIconHeight } from './constants'
+
+import InstallDropdownButton from './InstallDropdownButton'
+import { InstallationConfiguration } from './InstallationConfiguration'
+
+function InstalledActions({ installation, ...props }: any) {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <>
+      <Flex
+        align="center"
+        gap="medium"
+        {...props}
+      >
+        <InferredConsoleButton
+          primary
+          secondary={false}
+          text="Console"
+          flexGrow={1}
+        />
+        <Button
+          secondary
+          onClick={() => setOpen(true)}
+        >
+          <GearTrainIcon position="relative" top={4} />
+        </Button>
+      </Flex>
+      <InstallationConfiguration
+        open={open}
+        setOpen={setOpen}
+        installation={installation}
+      />
+    </>
+  )
+}
+
+function RepositorySideCar({ ...props }) {
```

Comment:
Why are you destructuring props like this?

Reply:
OH that's because it had some other props during dev. Let me fix that.